torchok/data/datasets/classification/classification.py

```diff
@@ -40,6 +40,7 @@ def __init__(self,
                  input_dtype: str = 'float32',
                  target_column: str = 'label',
                  target_dtype: str = 'long',
+                 weight_column: str = 'weight',
```

Comment:
Sampler is different entity, we need split dataset entity and sampler entity. This parameter can't be here.